frontend/css/spacing.css

```diff
@@ -0,0 +1,49 @@
+/* Setting up CSS variables */
+:root {
+    --right-padding: 1.5rem;
+    --small-gap: 1.5rem;
+    --large-gap: 2rem;
+    --btm-padding-sml: 0.1rem;
+    --btm-padding-sml2: 0.2rem;
+    --btm-padding-sml2: 0.2rem;
+    --large-padding: 2rem;
+}
+
+.grid {
+    gap: var(--large-gap);
+    --grid-padding-top: 1rem;
+    padding-top: var(--grid-padding-top);
+}
+
+/* Spacing for header section */
+.account {
+    padding-right: var(--right-padding);
+}
+
+.account-links {
+    gap: var(--small-gap);
```

Comment:
Sorry I should have mentioned this before. I would prefer to move this rule into where we specify `.account-links` in `layout.css`. 

So this file would preferably include only the spacing variables (and possibly some very generic classes - although I would probably place those classes in `layout.css` as well. And as that file grows, I would create separate files for each of the sections (`categories` for example) for the styles that are specific to that part of the page.